hq/conf/application.conf

```diff
@@ -27,10 +27,15 @@ stage = ${STAGE}
 
 host=${HOST}
 
+stack = security
+app = security-hq
+
 hq {
   # default to empty list if the config is missing (for now)
   account = []
   accounts = ${?AWS_ACCOUNTS}
+  organisation = guardian
+  snyk_token = ${SNYK_TOKEN}
```

Comment:
If the snyk token is a required config property what's parameter store being used for?

Reply:
It isn't any more, that should have been removed.